common/event/tasksreleasedevent.go

```diff
@@ -41,14 +41,23 @@ func (tr *TasksReleasedEvent) GetName() string {
 }
 
 func (tr *TasksReleasedEvent) GetEnvironmentId() uid.ID {
+	if tr == nil {
+		return ""
+	}
```

Comment:
This is all good & I fully agree, but were we triggering this kind of case? It might generally be better for the caller to also check whether the `TasksReleasedEvent` is `nil`, otherwise this condition stays silent. Same goes for other such funcs in this PR.